### PR TITLE
Update release note 1.2.0

### PR DESCRIPTION
**Pull request by yiyione**

### CODE REVIEW

**Comment by fanyangCS** [comment]
maybe we could include a upgrade guide?

**Comment by yiyione** [comment]
> maybe we could include a upgrade guide?

We already have a upgrade guide in [OpenPAI Handbook](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/upgrade-guide.html), I think users can still follow it.

**Comment by fanyangCS** [comment]
> > maybe we could include a upgrade guide?
> 
> We already have a upgrade guide in [OpenPAI Handbook](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/upgrade-guide.html), I think users can still follow it.

good. could you include the URL to upgrade guide in the release note?

**Comment by yiyione** [comment]
> good. could you include the URL to upgrade guide in the release note?

OK, Added.

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+  - New RestServer Arch: RestServer -> DB -> ApiServer (#4651)
+- Webportal
+  - Job list paging in server side
+  - Change job createdTime to submissionTime
+  - VC && Group experience for Admin (AAD Mode) (#4800)
+  - Support SKU count and SKU type in job submission page (#4796)
+  - Upgrade api/v1 code to api/v2 (#4704)
+  - Show "More Diagnostics" in job detail page (#4670)
+- Marketplace
+  - [Unified PAI protocol](https://github.com/microsoft/openpaimarketplace#49)
```
Comment (scarlett2018) [inline RELEASE_NOTE.md:15]: can we add issue link as the above for all the items? i.e. #4651, i.e. Job list paging in server side, etc.

**Review by scarlett2018 — CHANGES_REQUESTED** [review]

File: RELEASE_NOTE.md (hunk 1 above)
Comment (yiyione) [reply RELEASE_NOTE.md:15]: Added.

**Pull request merged**